src/style.css

```diff
@@ -6,23 +6,30 @@ body {
   box-sizing: border-box;
 }
 
+body {
+  background-image: url(img/bg5.jpg);
+  background-size: cover;
+}
+
 h1 {
   text-align: center;
   margin: 0 auto;
   margin-top: 20px;
+  color: #fff;
 }
 
 .container {
   width: 60%;
-  padding-top: 20px;
-  border: 3px solid #ec9b32;
+  padding-top: 30px;
   margin: auto;
   margin-top: 3rem;
   display: flex;
```

Comment:
- [ ] You have used `display: flex` 5 times. This falls under the category of bad practice because you are repeating yourself. I would recommend you create a class with a name for example d-flex and give the display:flex there and use it whenever you need it